blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,284 @@
+
+<Image src={require("./img/tabstops.webp").default} />
+
+<PCSX2PRLink authors="TellowKrinkle" prNums="12829,12830">
+  ### Improvements to PCSX2 related files handling on macOS
+</PCSX2PRLink>
+
+TellowKrinkle, our resident macOS housekeeper, has been working on adding file association to some of file that can be opened by PCSX2 such as game dumps (ISO, CSO, ZSO, CHD, etc), save states, GS Dumps, PNACH Patches, etc.
+
+<PCSX2PRLink authors="TellowKrinkle" prNums="13407">
+  ### PCSX2 on macOS Now Uses Qt 6.10
+</PCSX2PRLink>
+
+For a long time, macOS have been stuck on Qt 6.7 due to Qt 6.10 dropping older macOS version. It turns out Tellow discovered that its trivial to patch the compatibility for older macOS back to Qt 6.10, and the rest is history!
+
+<PCSX2PRLink
+  authors="TheLastRar"
+  prNums="13466,12864,13360,13555,13557,13577,12864"
+>
+  ### Improved Display Surface (And Wayland!)
+</PCSX2PRLink>
+
+We have enabled Wayland support by default since PCSX2 2.4.0 release and in this cycle, TheLastRar has yet again spearheaded the work to improve Wayland support even further. This time we're cutting out the crufts of technical debts and implement it the way it (Qt) was intended.
+
+For more technical details, previously the way we render into "surfaces" was to just rendering it to the widget directly. While this was the canonical method for older versions of Qt, the Qt maintainers mentioned that this was not recommended and a newer method was better suited. This has caused some issues especially with Wayland.
+
+Instead, this new method now uses a Qt Window instead of a Qt Widget, avoiding issues inherited by the previous method and allowing for the removal of a number of Wayland specific workarounds.
+
+#### Simplify how we bundle Wayland plugins ([#12864](https://github.com/PCSX2/pcsx2/pull/12864))
+
+`linuxdeploy` doesn't correctly deploy some Wayland dependencies with the Wayland client modules.
+
+Previously, we manually copied these dependencies into the AppImage, however, it was discovered that specifying the Wayland compositor module will copy the needed client dependencies.
+
+This allowed us to simplify our AppImage build script.
+
+<PCSX2PRLink authors="Gonzalosilvalde" prNums="12653">
+  ### New Gamma Adjustment Option
+</PCSX2PRLink>
+
+Gonzalosilvalde has worked on adding gamma adjustment to shade boost within the post processing options. Adjusting the gamma allows you to intensify the shadows while keeping the overall color balance intact.
+
+<Image src={require("./img/gamma.webp").default} />
+
+<PCSX2PRLink authors="SternXD,flavionm" prNums="12953,12698">
+  ### Adjustable Position for Various In-Game Overlays
+</PCSX2PRLink>
+
+SternXD and flavionm has worked on a feature where you can adjust the positions of all overlays inside PCSX2 (such as achievements and OSD notifications)
```

Comment:
Plurality, change has to have: "SternXD and flavionm have worked".

Missing period at the end of the sentence: "inside PCSX2 (such as achievements and OSD notifications)."